### PR TITLE
Fix API documentation and swagger

### LINKED ISSUE #1827: Fix API documentation and swagger
### Actual behavior
1. GET /transactions
    a. Swagger description not clear for address because there is senderAddress and recipientAddress
        https://github.com/LiskHQ/lisk-service/blob/development/services/gateway/apis/http-version3/methods/transactions.js#L30
    b. Nonce is only allowed if passed along with senderAddress. This is not mentioned in documentation and swagger.
        https://github.com/LiskHQ/lisk-service/blob/development/docs/api/version3.md#request-parameters-2

2. POST /transaction/dryrun
    a. Specify request body (flags like strict, skip verify and skip decode) in API documentation. We already specify it in swagger.

3. /blockchain/apps/meta/list
    a. Network regex wrong in API documentation.

4. /blockchain/apps/meta
    a. isDefault parameter is not clear in documentation.

5. /blockchain/apps/meta/tokens
    a. Network regex wrong in API documentation in `/blockchain/apps/meta/tokens/` API.

6. /blockchain/apps
   a. Search by name (change to chainName)
   b. Mention search supports case insensitive string.
   c. Update status description to mention unregistered as a supported value

7. /network/statistics
    a. 503 response not present in swagger.

8. /market/prices
    a. 503 response not present in swagger.
    b. Remove 400 bad request in swagger

9. /pos/unlocks
    a. Small grammatical corrections can be made on `validatorAddress` and `isLocked` properties description of posUnlocks swagger model

10. /pos/stakers
    a. swagger description of search does not mention if it is filtering the stakers or validator.

11. /export/transactions/
    a. Endpoint and response mentioned wrongly in API documentation

12. /export/download
    a. Endpoint and response mentioned wrongly in API documentation

### Expected behavior
Above mentioned inconsistencies should not exist

### Steps to reproduce
Check swagger and API documentation for above mentioned APIs

### Which version(s) does this affect? (Environment, OS, etc...)
Lisk Service v0.7.0-beta.5

### PR DESCRIPTION
**Pull request by vardan10**
### What was the problem?
This PR resolves #1827 #1828

### How was it solved?
- [x] Corrected min values in all APIs
- [x] GET /transactions
  - [x] a. Swagger description not clear for address because there is senderAddress and recipientAddress
  - [x] b. Nonce is only allowed if passed along with senderAddress. This is not mentioned in documentation and swagger.
- [x] POST /transaction/dryrun
  - [x] a. Specify request body (flags like strict, skip verify and skip decode) in API documentation. We already specify it in swagger.
- [x] /blockchain/apps/meta/list
  - [x] a. Network regex wrong in API documentation.
- [x] /blockchain/apps/meta
  - [x] a. isDefault parameter is not clear in documentation.
- [x] /blockchain/apps/meta/tokens
  - [x] a. Network regex wrong in API documentation in /blockchain/apps/meta/tokens/ API.
- [x] /blockchain/apps
  - [x] a. Search by name (change to chainName)
  - [x] b. Mention search supports case insensitive string.
  - [x] c. Update status description to mention unregistered as a supported value 
- [x] /network/statistics
  - [x] a. 503 response not present in swagger.
- [x] /market/prices
  - [x] a. 503 response not present in swagger.
  - [x] b. Remove 400 bad request in swagger 
- [x] /pos/unlocks
  - [x] a. Small grammatical corrections can be made on validatorAddress and isLocked properties description of posUnlocks swagger model
- [x] /pos/stakers
  - [x] a. swagger description of search does not mention if it is filtering the stakers or validator.
- [x] /export/transactions/
  - [x] a. Endpoint and response mentioned wrongly in API documentation 
- [x] /export/download
  - [x] a. Endpoint and response mentioned wrongly in API documentation

### How was it tested?
- [x] Local

### CODE REVIEW

File: docs/api/version3.md [hunk 1]
```diff
@@ -6283,18 +6284,25 @@ _Invalid parameter_
...
+https://service.lisk.com/api/v3/export/transactions?address=lskkje69szpgmfaefmbf7zvw7ghc6mamdvf9z3cpw
+```
+
+### File retrieval
+Returns transaction history
 
 #### Endpoints
 
-- HTTP GET `/api/v3/exports/{filename}`
+- HTTP GET `/api/v3/export/download/`
```
Comment (nagdahimanshu) [inline docs/api/version3.md:6298]:
```suggestion
- HTTP GET `/api/v3/export/download`
```

File: services/gateway/apis/http-version3/swagger/definitions/networkStatus.json [hunk 2]
```diff
@@ -30,7 +30,7 @@
                     "lastBlockID": {
                         "type": "string",
                         "format": "id",
-                        "minLength": 1,
+                        "minLength": 64,
                         "maxLength": 64,
                         "example": "01967dba384998026fe028119bd099ecf073c05c045381500a93d1a7c7307e5b",
                         "description": "Current block ID"
```
Comment (nagdahimanshu) [inline services/gateway/apis/http-version3/swagger/definitions/networkStatus.json:36]:
```suggestion
                        "description": "Current block ID."
```

File: services/gateway/apis/http-version3/methods/market.js [hunk 3]
```diff
@@ -43,8 +43,13 @@ module.exports = {
...
 				},
 			},
+			503: {
+				description: 'Service Unavailable',
+				schema: {
+					$ref: '#/definitions/serviceUnavailable',
+				},
+			},
 		};
-		Object.assign(marketPricesSchema[this.swaggerApiPath].get.responses, response);
```
Comment (priojeetpriyom) [inline services/gateway/apis/http-version3/methods/market.js:47]: should't this api return bad request if called with invalid params?

**Review by nagdahimanshu — APPROVED** [review]
LGTM 👍

File: services/gateway/apis/http-version3/methods/market.js (hunk 3 above)
Comment (vardan10) [reply services/gateway/apis/http-version3/methods/market.js:47]: Applied the suggested change to take care of unwanted params

**Review by priojeetpriyom — APPROVED** [review]

**Pull request merged**